playbook/app/pb_kits/playbook/pb_stat_value/stat_value.rb

```diff
@@ -4,10 +4,10 @@ module Playbook
   module PbStatValue
     class StatValue < Playbook::KitBase
       prop :unit
-      prop :value, type: Playbook::Props::Number
+      prop :value, type: Playbook::Props::Numeric
```

Comment:
This is the main fix here. Without this type change, passing this kit a decimal/float would fail with a type error.